hypothesis/lms
# Add real API calls to get YouTube video info in YouTubePicker

This PR adds real API calls to fetch YouTube video metadata and thumbnail.

[Grabación de pantalla desde 2023-06-01 11-01-31.webm](https://github.com/hypothesis/lms/assets/2719332/ebfe7092-5810-4ae6-b693-e7c426d076d2)

### Testing steps

1. Check out this branch
2. Go to https://hypothesis.instructure.com/courses/125 (this instance has YouTube enabled)
3. Try to edit/create a course on that instance
4. Select YouTube video, and introduce a valid YouTube URL in the text field.
    * The video details should be loaded (thumbnail, video title and channel name).
    * Changing the URL to other videos should load the new details.
5. Change the URL to one that fulfils the format, but with an invalid ID
    * An error should be displayed.

### Next steps

- Handle transcript language selection when English not present.
- Consider displaying an error when selecting an age-restricted video.
